sklearn/manifold/t_sne.py

```diff
@@ -629,11 +646,9 @@ class TSNE(BaseEstimator):
     >>> X = np.array([[0, 0, 0], [0, 1, 1], [1, 0, 1], [1, 1, 1]])
     >>> model = TSNE(n_components=2, random_state=0)
     >>> np.set_printoptions(suppress=True)
-    >>> model.fit_transform(X) # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
-    array([[ 0.00017619,  0.00004014],
-           [ 0.00010268,  0.00020546],
-           [ 0.00018298, -0.00008335],
-           [ 0.00009501, -0.00001388]])
+    >>> X_embedded = model.fit_transform(X)
+    >>> X_embedded.shape
+    (4, 2)
```

Comment:
The range of values may be of interest to the user, and worth outputting them to illustrate.